data/json/mapgen/homeimprovement_superstore.json

```diff
@@ -90,478 +95,344 @@
+        "**********?zzzzzzzzzzzzz????????zzzzzzzzzzzzzz?*",
+        "**********?zzzzzzzzzzzzz?******?zzzzzzzzzzzzzz?*",
+        "**********???????????????******????????????????*",
+        "*************************************ZZZZZZZZZz*",
+        "*************************************ZZZZZZZZZz*",
+        "*************************************zzzzzzzzzz*",
+        "************************************************",
+        "************************************************",
+        "************************************************",
+        "************************************************",
+        "************************************************",
+        "************************************************",
+        "************************************************",
+        "************************************************",
+        "************************************************",
+        "************************************************",
+        "************************************************",
+        "************************************************",
+        "************************************************"
       ],
-      "terrain": {
-        "$": "t_floor",
-        "&": "t_floor",
-        "+": "t_door_glass_c",
-        ",": "t_pavement_y",
-        "-": "t_wall",
-        ".": "t_floor",
-        "C": "t_floor",
-        "T": "t_floor",
-        "W": "t_window",
-        "^": "t_floor",
-        "_": "t_pavement",
-        "c": "t_floor",
-        "r": "t_floor",
-        "s": "t_floor",
-        "t": "t_floor",
-        "|": "t_wall"
-      },
-      "furniture": {
-        "$": "f_vending_c",
-        "C": "f_chair",
-        "T": "f_table",
-        "^": "f_indoor_plant",
-        "c": "f_counter",
-        "r": "f_rack",
-        "s": "f_sink",
-        "t": "f_toilet"
+      "palettes": [ "home_improvement_palette" ],
+      "items": {
+        "d": { "item": "office", "chance": 45, "repeat": [ 1, 2 ] },
+        "D": { "item": "hardware_trash", "chance": 40, "repeat": [ 1, 3 ] },
+        "Q": { "item": "bed", "chance": 25, "repeat": [ 1, 2 ] },
+        "V": { "item": "vending_drink_home_impr", "chance": 85, "repeat": [ 5, 8 ] },
+        "1": { "item": "paint", "chance": 65 },
+        "2": { "item": "carpet", "chance": 65 },
+        "3": [ { "item": "farming_seeds", "chance": 40, "repeat": [ 1, 2 ] }, { "item": "farming_tools", "chance": 15 } ],
+        "4": { "item": "elec_parts", "chance": 25, "repeat": [ 1, 3 ] },
+        "5": { "item": "hardware_clothing", "chance": 25, "repeat": [ 1, 3 ] },
+        "6": [ { "item": "hand_tools", "chance": 25, "repeat": [ 1, 3 ] }, { "item": "power_tools", "chance": 15 } ],
+        "7": [ { "item": "farming_seeds", "chance": 40, "repeat": [ 1, 2 ] }, { "item": "farming_tools", "chance": 15 } ],
+        "8": { "item": "raw_materials", "chance": 35 },
+        "9": { "item": "raw_materials", "chance": 35 },
```

Comment:
You could add many more repeats to all these store stock items, like 5-10. This isn't the sort of store that should start the game heavily looted.